knowledge-base/buttongroup-mixed-selection-mode.md

```diff
@@ -0,0 +1,90 @@
+---
+title: ButtonGroup Mixed Selection Mode
+description: How to create a mix between Single and Multiple selection modes in Blazor ButtonGroup.
+type: how-to
+page_title: ButtonGroup Mixed Selection Mode
+slug: buttongroup-kb-mixed-selection-mode
+position: 
+tags: buttongroup, mixed, button, selection, mode, third, single, multiple
+ticketid: 1585768
+res_type: kb
+---
+
+## Environment
+
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>ButtonGroup for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+How to create a third selection mode that is a mix between `Single` and `Multiple`?
```

Comment:
The customer description is unclear and we should add one more. It should sound like "how to force at least one selected ButtonGroup Button", "minimum one selecteed button", etc.